src/discordcr/websocket.cr

```diff
@@ -33,10 +61,21 @@ module Discord
       )
     end
 
+    def on_binary(&handler : Packet ->)
+      @websocket.on_binary do |binary|
+        io = IO::Memory.new(binary)
+        Zlib::Reader.open(io) do |reader|
+          payload = Packet.from_json(reader)
+          @logger.debug "[WS IN] (compressed, #{binary.size}) #{payload.to_json}"
```

Comment:
This needs to be followed by `if @logger.debug?` (see its sister line in `on_message`)